Lib/fontTools/feaLib/parser.py

```diff
@@ -524,46 +524,44 @@ def parse_glyph_pattern_(self, vertical):
                 )
             return (prefix, glyphs, lookups, values, suffix, hasMarks)
 
-    def parse_chain_context_(self):
+    def parse_ignore_glyph_pattern_(self, sub):
         location = self.cur_token_location_
         prefix, glyphs, lookups, values, suffix, hasMarks = self.parse_glyph_pattern_(
             vertical=False
         )
-        chainContext = [(prefix, glyphs, suffix)]
-        hasLookups = any(lookups)
+        if any(lookups):
+            raise FeatureLibError(
+                f'No lookups can be specified for "ignore {sub}"', location
+            )
+        if not hasMarks:
+            error = FeatureLibError(
+                f'Ambiguous "ignore {sub}", there should be least one marked glyph',
+                location,
+            )
+            log.warning(str(error))
```

Comment:
no need to make a FeatureLibError exception if the only thing you need is str(error), just pass the string as log.warning message

Reply:
This is to show the location with the warning (this is used elsewhere in the file, but if there is a better way, I can use it).

Reply:
oh right, that's ok then.